packages/react-components/react-alert/Spec.md

```diff
@@ -1,42 +1,75 @@
-# @fluentui/react-alert Spec
+# Alert
 
 ## Background
 
-_Description and use cases of this component_
+An Alert displays a brief, important message to attract a user's attention without interrupting their current task.
 
 ## Prior Art
 
-_Include background research done for this component_
-
-- _Link to Open UI research_
-- _Link to comparison of v7 and v0_
-- _Link to GitHub epic issue for the converged component_
+- [Alert - Open UI](https://open-ui.org/components/toast.research)
+- [Convergence epic issue #22579](https://github.com/microsoft/fluentui/issues/22579)
 
 ## Sample Code
 
-_Provide some representative example code that uses the proposed API for the component_
+Default Alert
+
+```
+import { Alert } from '@fluentui/react-northstar'
+
+const AlertExample = () => <Alert content="This is a default alert" />
+```
+
+Success Alert
+
+```
+import { Alert } from '@fluentui/react-northstar'
+
+const AlertExample = () => <Alert intent="success" content="This is a success alert" />
+```
+
+Dismissible Error Alert
 
-## Variants
+```
+import { Alert } from '@fluentui/react-northstar'
 
-_Describe visual or functional variants of this control, if applicable. For example, a slider could have a 2D variant._
+const AlertExample = () => <Alert intent="error" content="This is an error alert" dismissible/>
+```
+
+InfoAvatar Alert
+
+```
+import { Alert, Avatar } from '@fluentui/react-northstar'
+
+<Alert
+    intent="infoAvatar"
+    content="This is an avatar alert"
+    avatar={<Avatar name="John Doe" />
+  />
+```
 
 ## API
 
 _List the **Props** and **Slots** proposed for the component. Ideally this would just be a link to the component's `.types.ts` file_
 
+TODO - Add interfaces to the type file & copy here
```

Comment:
Instead of copying the interface, you can link to the types file (we're currently doing this change in other components)